auslib/admin/views/history.py

```diff
@@ -1,27 +1,17 @@
-import difflib
 import json
 
-from sqlalchemy.sql.expression import null
-
-from flask import Response
-
-from auslib.global_state import dbo
-from auslib.admin.views.base import AdminView
+from flask import Response, jsonify, request
+from sqlalchemy import and_
+from auslib.admin.views.base import AdminView, HistoryAdminView
 
 
 class FieldView(AdminView):
-    """/view/:type/:id/:field"""
```

Comment:
I see that FieldView is only ever used by ReleaseDiffView, and it makes me wonder if we should just kill it (and move the small bit of necessary logic into ReleaseDiffView. What do you think?